README.md

```diff
@@ -18,17 +18,14 @@ Simple example:
 
 
 const templater = require('anytv-templater');
+const i18n = require('anytv-i18n');
```

Comment:
i18n version in package.json is 1.0.1. Isn't the usage in the update readme for 2.X.X?